### PR TITLE
Fix : Search Screen Spacing & Input Cursor Position

### PR DESCRIPTION
**Pull request by omjogani**
#532 Fixed...

### Files Modified:
- lib/Screens/Search/search.dart

Modification Description: Added Vertical Padding of 10 to separate out Chips and Text (Mentioned in the Issue), Added code to change cursor position to the end of the chosen text from the chip.

### Final Output

https://user-images.githubusercontent.com/72139914/224481573-0d04c0c7-776e-49f0-b6d4-babd5a7d0f7a.mp4

Let me know in case of any issues or queries.
Thanks
-Om Jogani

### CODE REVIEW

File: lib/Screens/Search/search.dart [hunk 1]
```diff
@@ -210,6 +211,7 @@ class _SearchPageState extends State<SearchPage> {
                                                     .toString()
                                                     .trim();
                                                 controller.text = query;
+                                                controller.selection = TextSelection.fromPosition(TextPosition(offset: query.length));
```
Comment (BrightDV) [inline lib/Screens/Search/search.dart:214]: Just a small formatting issue.
```suggestion
                                                controller.selection = TextSelection.fromPosition(
                                                	TextPosition(offset: query.length),
                                                );
```

File: lib/Screens/Search/search.dart [hunk 2]
```diff
@@ -289,6 +291,7 @@ class _SearchPageState extends State<SearchPage> {
                                                               .trim();
                                                           controller.text =
                                                               query;
+                                                          controller.selection = TextSelection.fromPosition(TextPosition(offset: query.length));
```
Comment (BrightDV) [inline lib/Screens/Search/search.dart:294]: Same here.
```suggestion
                                                          controller.selection = TextSelection.fromPosition(
                                                          	TextPosition(offset: query.length),
                                                          );
```

**Review by BrightDV — COMMENTED** [review]
There are two formatting issue that you have to fix before it gets merged.  
Also, you can run `dart format .` in your directory to fix them.

**Comment by omjogani** [comment]
@BrightDV Done, Code is Formatted!

**Comment by Sangwan5688** [comment]
Please add the fix for yt search as well

**Comment by omjogani** [comment]
@Sangwan5688 I think it's not necessary there because there is no chip or anything which directly put text in the input field, There is a suggestion that directly searches and fetches results with focus out.

if it's there somewhere, and I haven't discovered it, let me know...

Thanks
-Om Jogani

**Comment by Sangwan5688** [comment]
@omjogani take the latest master merge. You'll find it there

**Comment by omjogani** [comment]
@Sangwan5688 I've fixed it in YT Search as well...

**Comment by Sangwan5688** [comment]
merged via another PR

**Comment by Sangwan5688** [comment]
https://github.com/Sangwan5688/BlackHole/pull/541

**Pull request closed without merging**